### PR TITLE
[R4R]-[mainnet]feat: update mainnet deploy config

### PR DESCRIPTION
**Pull request by Tri-stone**

### CODE REVIEW

File: packages/contracts-bedrock/deploy-config/mantle-mainnet.json [hunk 1]
```diff
@@ -2,42 +2,42 @@
   "l1ChainID": 1,
   "l2ChainID": 5000,
   "l2BlockTime": 2,
-  "maxSequencerDrift": 300,
-  "sequencerWindowSize": 200,
-  "channelTimeout": 120,
+  "maxSequencerDrift": 600,
+  "sequencerWindowSize": 3600,
+  "channelTimeout": 300,
   "p2pSequencerAddress": "0x0000000000000000000000000000000000000000",
```
Comment (Tri-stone) [inline packages/contracts-bedrock/deploy-config/mantle-mainnet.json:8]: p2pSequencerAddress？

File: packages/contracts-bedrock/deploy-config/mantle-mainnet.json [hunk 2]
```diff
@@ -2,42 +2,42 @@
...
+  "baseFeeVaultRecipient": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "l1FeeVaultRecipient": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "sequencerFeeVaultRecipient": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "proxyAdminOwner": "0x4e59e778a0fb77fBb305637435C62FaeD9aED40f",
+  "finalSystemOwner": "0x4e59e778a0fb77fBb305637435C62FaeD9aED40f",
+  "portalGuardian": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "controller": "0x2A2954F3989a83Cc43DD58B0f038D5F276f21333",
+  "finalizationPeriodSeconds": 604800,
+  "numDeployConfirmations": 2,
   "fundDevAccounts": true,
```
Comment (Tri-stone) [inline packages/contracts-bedrock/deploy-config/mantle-mainnet.json:32]: fundDevAccounts？

File: packages/contracts-bedrock/deploy-config/mantle-mainnet.json [hunk 3]
```diff
@@ -2,42 +2,42 @@
...
-  "l2GenesisBlockBaseFeePerGas": "0x3B9ACA00",
-  "gasPriceOracleOwner": "0x0000000000000000000000000000000000000000",
-  "gasPriceOracleTokenRatio": 4500,
-  "gasPriceOracleOverhead": 2100,
-  "gasPriceOracleScalar": 1000000,
+  "l2GenesisBlockBaseFeePerGas": "0x1312D00",
```
Comment (Tri-stone) [inline packages/contracts-bedrock/deploy-config/mantle-mainnet.json:33]: baseFee set to 0.02gwei

File: packages/contracts-bedrock/deploy-config/mantle-mainnet.json [hunk 4]
```diff
@@ -2,42 +2,42 @@
...
   "l1StartingBlockTag": "latest",
   "l1MantleToken": "0x3c3a81e81dc49a522a592e7622a7e711c06bf354",
 
-  "l2OutputOracleProposer": "0x0000000000000000000000000000000000000000",
-  "l2OutputOracleChallenger": "0x0000000000000000000000000000000000000000",
-  "l2GenesisBlockGasLimit": "0x4000000000000",
-  "l1BlockTime": 3,
+  "l2OutputOracleProposer": "0x6667961f5e9C98A76a48767522150889703Ed77D",
+  "l2OutputOracleChallenger": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "l2GenesisBlockGasLimit": "0xE8D4A51000",
```
Comment (HaoyangLiu) [inline packages/contracts-bedrock/deploy-config/mantle-mainnet.json:20]: `1000000000000` ? Not `400000000000`?

**Review by HaoyangLiu — DISMISSED** [review]

**Review by HaoyangLiu — APPROVED** [review]

File: packages/contracts-bedrock/deploy-config/mantle-mainnet.json [hunk 5]
```diff
@@ -2,42 +2,42 @@
...
+  "fundDevAccounts": false,
+  "l2GenesisBlockBaseFeePerGas": "0x1312D00",
+  "gasPriceOracleOwner": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "gasPriceOracleTokenRatio": 4000,
+  "gasPriceOracleOverhead": 188,
+  "gasPriceOracleScalar": 50000,
   "governanceTokenSymbol": "MNT",
   "governanceTokenName": "Mantle",
-  "governanceTokenOwner": "0x0000000000000000000000000000000000000000",
+  "governanceTokenOwner": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
```
Comment (shidaxi) [inline packages/contracts-bedrock/deploy-config/mantle-mainnet.json:40]: owner is `[0x4e59e778a0fb77fBb305637435C62FaeD9aED40f]`

File: packages/contracts-bedrock/deploy-config/mantle-mainnet.json [hunk 6]
```diff
@@ -2,42 +2,42 @@
...
+  "sequencerFeeVaultRecipient": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "proxyAdminOwner": "0x4e59e778a0fb77fBb305637435C62FaeD9aED40f",
+  "finalSystemOwner": "0x4e59e778a0fb77fBb305637435C62FaeD9aED40f",
+  "portalGuardian": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
+  "controller": "0x2A2954F3989a83Cc43DD58B0f038D5F276f21333",
+  "finalizationPeriodSeconds": 604800,
+  "numDeployConfirmations": 2,
+  "fundDevAccounts": false,
+  "l2GenesisBlockBaseFeePerGas": "0x1312D00",
+  "gasPriceOracleOwner": "0x2F44BD2a54aC3fB20cd7783cF94334069641daC9",
```
Comment (shidaxi) [inline packages/contracts-bedrock/deploy-config/mantle-mainnet.json:34]: `0x2A2954F3989a83Cc43DD58B0f038D5F276f21333` then transfer to `0xbA9026Cf9d17cFA6DEE95d59aB46b60A56c1e505`

**Review by shidaxi — APPROVED** [review]

**Review by HaoyangLiu — APPROVED** [review]

**Review by shidaxi — APPROVED** [review]

**Review by byteflyfunny — APPROVED** [review]

**Pull request merged**